#request
float64[] trocarAngles
---
#response
geometry_msgs/Pose trocar

